CHANGELOG.md

```diff
@@ -4,9 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.8.3] - 2022-2-23
```

Comment:
Wrong date. Also there's no link at the bottom to go with [1.8.3]

Reply:
:banana: Whoops been a while since I've done gem work! Thanks!